MinPlatformPkg/Test/Library/PlatformConfigCheckLibNull/DxePlatformConfigCheckLibNull.c

```diff
@@ -0,0 +1,28 @@
+/** @file
+
+  Platform configuration check library NULL implementation for TestPointCheckLib
+
+  Copyright (c) Microsoft Corporation. All rights reserved
```

Comment:
I suggest adding a period to the end (here and other files) so this exactly matches potential searches.

```suggestion
  Copyright (c) Microsoft Corporation. All rights reserved.
```